pkgs/applications/radio/m17-cxx-demod/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub, cmake, pkg-config, boost, codec2 }:
+
+stdenv.mkDerivation rec {
+  pname = "m17-cxx-demod";
+  version = "2.3";
+
+  src = fetchFromGitHub {
+    owner = "mobilinkd";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-mvppkFBmmPVqvlqIqrbwGrOBih5zS5sZrV/usEhHiws=";
+  };
+
+  nativeBuildInputs = [ cmake pkg-config boost ];
+  buildInputs = [ codec2 ];
+
+  meta = with lib; {
+    description = "M17 Demodulator in C++";
+    homepage = "https://github.com/mobilinkd/m17-cxx-demod";
+    license = licenses.gpl3Only;
+    platforms = platforms.unix;
+    maintainers = with maintainers; [ astro ];
```

Comment:
We could also put that under c3d2 if you like.

Reply:
What does "c3d2" mean?

Reply:
We collectively maintain packages that are used in our shared infrastructure. See #178612